code/Spikes/Class_WiFiService/WiFiService.cpp

```diff
@@ -0,0 +1,131 @@
+
+#include <Arduino.h>
+#include "WiFiService.h"
+#include <SoftwareSerial.h>
+
+
+// Create SoftwareSerial object
+SoftwareSerial mySerial(10, 11); // RX, TX
+
+//Constructor
+WiFiService::WiFiService()
+{
+  StringComplete = false;
+  StringStarted = false;
+  rxString.reserve(50);  
+  rxString = "Test";
+  SerialChar = "";
+  StringLength = 0;
+  mySerial.begin(9600);   // Set data rate for the SW serial port
+  while (!mySerial) {;}   // wait for SW serial port to connect.
+}
+
+void WiFiService::Init(bool mode)
+{
+  //Mode "Wait_for_Start_Character": Does Bidirectional communication until it sees '['-Character, then Mode "BuildString" is entered in teh next Main Loop.
+  if (mode == true && !StringComplete)        {Wait_for_Start_Character();  HWtoSWSerial();}  
```

Comment:
Du hast schon Recht es gibt keinen Grund es auszuführen, aber ich habe auch keinen Grund gefunden es zu beenden.
HW to SW Serial ist ja die Kommunikation von Arduino IDE->USB->Arduino Board->WIFiModul. Werden wir später nicht brauchen, aber ich hab es einfach mal drin gelassen, damit man immer vom PC aus Nachrichten an die App schicken kann.